lxl-web/src/routes/(app)/[[lang=lang]]/[fnurgel=fnurgel]/Holdings.svelte

```diff
@@ -40,10 +40,19 @@
 	function missingAtLeastOneLinkToItem() {
 		return bibIds.find((id) => !hasLinksToItem(id.bibId) && !hasLoanReserveLink(id.bibId));
 	}
+
+	function hasOpeningHoursEtc() {
+		const id = bibIds.at(0);
+		return (
+			id &&
+			(linksByBibId[id.bibId]?.[holder.sigel]?.[BibDb.OpeningHours] ||
+				linksByBibId[id.bibId]?.[holder.sigel]?.[BibDb.Address])
+		);
+	}
 </script>
 
 <li class="border-neutral text-sm not-last:border-b">
-	<h2 class="holder-label mt-4">
+	<h2 class="holder-label mt-4 line-clamp-1">
```

Comment:
`line-clamp-1` added here and `line-clamp-2` included in `holder-label class`...

Reply:
Woops!